frontend/src/v5/ui/components/viewer/cards/cardFilters/filterForm/filterFormValues/rangeInput/useRangeEffect.ts

```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright (C) 2024 3D Repo Ltd
+ *  Copyright (C) 2025 3D Repo Ltd
```

Comment:
There's a white page being thrown by dates
steps to reproduce
- open the ticket filters modal.
- select a date property to filter
- change the criteria to "In Range"
- select the date in the left field (leave the right one empty)
- click on "back" button
- select another date property to filter (it could be the same)
- the page goes blank